source/reference/about-mongosync.txt

```diff
@@ -0,0 +1,157 @@
+~~~~~~~~~~~
+
+To start the synchronization process, call the :ref:`start <c2c-api-start>` 
+endpoint. 
+
+The ``start`` endpoint begins the ``RUNNING`` state, which is when ``mongosync`` 
+syncs data from the source cluster to the destination cluster. Then, if you 
+make subsequent writes to the source cluster, ``mongosync`` applies the 
+operations to the destination cluster. 
+
+For every source collection that syncs, 
+``mongosync`` creates a corresponding collection on the destination. 
+
+``mongosync`` temporarily alters some of the replicated collection
+characteristics in the destination cluster. It changes these destination cluster
+characteristics back to the source cluster characteristics during the COMMIT 
+state. For more information, see :ref:`Commit Alters <c2c-commit-alters>`.
+
+The ``RUNNING`` state includes two phases:
+
+- :ref:`Collection copy <c2c-collection-copy>`
+
+- :ref:`Change event application <c2c-cea>`
+
+.. _c2c-collection-copy:
+
+Collection Copy
+'''''''''''''''
+
+During the collection copy phase, ``mongosync`` executes the initial data 
+transfer from the source cluster to the destination 
+cluster. ``mongosync`` performs this data transfer by splitting the 
+source collections into partitions and copying the partitions to their 
+corresponding destination collections.
+
+.. _c2c-cea:
+
+Change Event Application
+''''''''''''''''''''''''
+
+The change event application (CEA) phase starts after the collection copy phase
+ends. 
+
+During the CEA phase, ``mongosync`` continuously 
+updates the destination cluster when you make changes to the source cluster. 
+
+To make continuous updates to the destination cluster, ``mongosync`` opens a 
+:ref:`change stream <changeStreams>` on the source cluster and applies the
+:ref:`change events <change-events>` it receives to the destination cluster. 
+
+When the CEA phase begins, ``mongosync`` first applies the change events 
+that are received during the collection copy phase before moving on to the 
+events that are received during the CEA phase. 
+
+Pausing Sync
+~~~~~~~~~~~~
+
+To pause the sync process and enter the :ref:`PAUSED <c2c-state-paused>` state,
+call the :ref:`pause <c2c-api-pause>` endpoint. 
+
+To resume a paused sync process and return ``mongosync`` to the 
+``RUNNING`` state, call the :ref:`resume <c2c-api-resume>` endpoint. 
+
+Finalizing Sync
+~~~~~~~~~~~~~~~
+
+To finalize the sync between the source and destination clusters, 
+call the :ref:`commit <c2c-api-commit>` endpoint. 
+
+The ``commit`` endpoint starts the :ref:`COMMITTING <c2c-state-committing>` 
+state, which is when ``mongosync`` stops continuous sync between the source and 
+destination clusters. 
+
+When the ``COMMITTING`` state begins, ``mongosync`` retrieves the timestamp of 
+the most recent operation on the source cluster. 
+
+If there are no errors, ``mongosync`` sends a success message and exits CEA by 
+processing events up until the commit timestamp. It also restores collection 
```

Comment:
[nit] [style guide](https://www.mongodb.com/docs/meta/style-guide/writing/use-pronouns-carefully/#it) - clarify pronouns and antecedents. "It" is a little ambiguous here on whether it's referring to `mongosync` or the committing state.

Reply:
Fixing now!